care/facility/api/serializers/prescription.py

```diff
@@ -22,18 +22,18 @@ class Meta:
 class PrescriptionSerializer(serializers.ModelSerializer):
     id = serializers.UUIDField(source="external_id", read_only=True)
     prescribed_by = UserBaseMinimumSerializer(read_only=True)
-    last_administered_on = serializers.SerializerMethodField()
+    last_administration = serializers.SerializerMethodField()
```

Comment:
This can be a `MedicineAdministrationSerializer` itself right instead of a `SerializerMethodField`?